### DISCUSSION TITLE
Request to support Alpine for `runs-on`

### DISCUSSION THREAD

**Discussion by rinarakaki** [community/community#53184] [Actions]
### Select Topic Area

General

### Body

Supporting Alpine will be quite valuable as it is more and more widely used and has unique build stack (musl).

**Comment by airtower-luna** [comment]
You can [use a job container](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idcontainer) with an Alpine image to run your build on Alpine. I [use that to test with a few different distros](https://github.com/airtower-luna/mod_gnutls/blob/95d83210d449f632a36f343b3ce1b2f8fd60489d/.github/workflows/build.yaml#L19-L26), including Alpine.
